app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -57,18 +60,27 @@ echo '#'
 echo '# Adding repository'
 echo '#'
 
-if [ -f /etc/redhat-release ]; then
-  cat << EOF > /etc/yum.repos.d/foreman_registration.repo
+
+if [ x$PKG_MANAGER = xdnf -o x$PKG_MANAGER = xyum -o x$PKG_MANAGER = xzypper ]; then
+  cat << EOF > /tmp/foreman_registration.repo
 [foreman_register]
 name=foreman_register
 baseurl=<%= shell_escape @repo %>
```

Comment:
In the UI Form [0] and API [1] are examples for the Red Hat Family and Debian family.
Can we add examples also for the Suse/SLES family?

Note: Would be also nice to update the docs [2], we don't have example there at all. 

[0] https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/Repository.js
[1] https://github.com/theforeman/foreman/blob/develop/app/controllers/api/v2/registration_commands_controller.rb#L22
[2] https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#Customizing_the_Registration_Templates_managing-hosts

Reply:
Done. I will ask my colleage to have a look at the docs.